### PR TITLE
Initial draft of goals

### PR DESCRIPTION
**Pull request by benfurber**
I've had a few discussions with people about having sone criteria to track my progress through the apprenticeship and a clear set of statements that can be used to assess my promotion to Developer.

This is my first draft at those statements and I'd love your feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,29 @@
...
+* I can build apps in React Native
+* I understand how React Native works
+* I understand how the web works
+* I understand how databases work
+
+*I'm a consultant*
+* I communicate clearly with clients
+* I can clearly explain to others where/when thoughtbot can add value
+* I understand how technical/design decisions will impact clients
+* My teams build software clients’ love
```
Comment (purinkle) [inline README.md]: Would it be helpful to [make these checklists]?

[make these checklists]: https://blog.github.com/2014-04-28-task-lists-in-all-markdown-documents/

Comment (benfurber) [reply README.md]: @purinkle - Can do though I think the real progress will be tracked in the app when built. Maybe as a separate document I'll do an initial assessment - like you were asking me for earlier.

File: docs/initial-assessment.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+* I can build apps in React Native (1.5/5 - could confidently build a frontend only app)
+* I understand how React Native works (2/5 - could explain the basics)
+* I understand how the web works (3/5)
+* I understand how databases work (2/5)
+
+*I'm a consultant*
+* I communicate clearly with clients (3/5 - I think my Product background should set me up well)
+* I can clearly explain to others where/when thoughtbot can add value (1/5)
+* I understand how technical/design decisions will impact clients (2.5/5 - PO experience)
+* My teams build software clients’ love (0.5/5 - Contributions will grow)
```
Comment (benfurber) [inline docs/initial-assessment.md]: @purinkle

Comment (purinkle) [reply docs/initial-assessment.md]: All looks good to me.

I'd merge this and add anything further in future PRs.

We should talk about what you want to work on first and how we are going to track this.

**Pull request merged**